.github/workflows/iroha2-dev.yml

```diff
@@ -1,29 +1,21 @@
-name: I2::Dev::Deploy
+name: I2::Dev::Publish
 
 on:
   push:
     branches: [iroha2-dev]
 
 env:
   CARGO_TERM_COLOR: always
-  REPOSITORY_URL: '${{ github.server_url }}/${{ github.repository }}.git'
 
 # Note jobs in this workflow are run on `push` meaning that there's
 # no point in burning our AWS self-hosted runners' time. Hence
 # `ubuntu-latest` and not `[self-hosted, Linux]`.
 jobs:
-  deploy:
+  publish:
```

Comment:
```suggestion
  dockerhub:
```

Reply:
Similarly replace `print_telemetry` with `telemetry`.

Reply:
I'd take a more explicit way: `publish-image`, `archive-binaries_and_schema`, and `print-telemetry`